clang/include/clang/ASTMatchers/ASTMatchers.h

```diff
@@ -1090,6 +1090,7 @@ AST_POLYMORPHIC_MATCHER_P2(
 AST_POLYMORPHIC_MATCHER_P(
     templateArgumentCountIs,
     AST_POLYMORPHIC_SUPPORTED_TYPES(ClassTemplateSpecializationDecl,
+                                    VarTemplateSpecializationDecl, FunctionDecl,
```

Comment:
It'd probably be good to update the example code above. Also, can you add test coverage to `clang/unittests/ASTMatchers/ASTMatchersNarrowingTest.cpp`?